fluentd/configs.d/openshift/filter-retag-journal.conf

```diff
@@ -67,10 +67,12 @@
   rewriterule4 CONTAINER_NAME ^k8s_[^_]+_[^_]+_openshift-(.+)_ kubernetes.journal.container._openshift-$1_
   # mark logs from openshift namespace for processing as k8s logs but stored as system logs
   rewriterule5 CONTAINER_NAME ^k8s_[^_]+_[^_]+_openshift_ kubernetes.journal.container._openshift_
+  # mark eventrouter events
+  rewriterule6 CONTAINER_NAME ^k8s_[^_]+_kube-eventrouter-[^_]+_ kubernetes.journal.container._kube-eventrouter_
```

Comment:
sorry - looks like the pod name is this:
```
oc get -n default pods
NAME                          READY     STATUS    RESTARTS   AGE
logging-eventrouter-1-nvkvw   1/1       Running   0          1h
```
so the pattern should be `^k8s_[^_]+_logging-eventrouter-[^_]+_

Reply:
Also - make the tag `kubernetes.journal.container.kubernetes-event` as per @jcantrill 's suggestion

Reply:
Thanks for the reminder.  Done!

This suggestion by @jcantrill is not taken care yet...
> Consider actually extending the other filter and calling its filter method if needed.

Does this mean instead of adding filter_elasticsearch_genid_ext, we are having filter_elasticsearch_genid in our code and make it something like this?
```
module Fluent
  class ElasticsearchGenidFilter < Filter
    ....
  end
  class ElasticsearchGenidExtFilter < ElasticsearchGenidFilter
    ....
  end
end
```
And could you tell me where I can take advantage of calling `super`?
> Call super on the base class once you extend what already exists